static/js/stat_var_hierarchy/node_header.tsx

```diff
@@ -64,6 +67,12 @@ export class StatVarHierarchyNodeHeader extends React.Component<
         <span className="title">
           {this.props.title}
           {showCount && " (" + this.props.count + ")"}
+          {showAsterisk && (
+            <>
+              {" "}
+              (<sup>*</sup>)
```

Comment:
remove parenthesis? Just have * could work too

Reply:
I think without the parenthesis, the * might be a little hard to notice unless you're looking for it ..

this is what it looks like without the parenthesis. What do you think?
<img width="1774" alt="Screen Shot 2021-06-24 at 4 17 25 PM" src="https://user-images.githubusercontent.com/69875368/123343994-c023ef80-d507-11eb-9d1c-16d296ae55e2.png">

Reply:
Instead of asterisk, maybe we could do a checkmark like this

@beets @shifucun 

<img width="1777" alt="Screen Shot 2021-06-24 at 4 38 05 PM" src="https://user-images.githubusercontent.com/69875368/123345363-be0f6000-d50a-11eb-855b-189463385c31.png">

Reply:
I like the last option, looks very nice!!

Reply:
+1 to the last option! thanks for trying out a few things!

Reply:
done, updated!